### ISSUE TITLE
GNSS Fix Type in Geopackage Log

### ISSUE THREAD

**Issue by geopackix** [qgis/QGIS#57919]
### Feature description

The current GNSS Geopackage Log contains a set of columns but the fix type (Single, DGPS, Float, Fix) is missing. (NMEA GGA GPS Quality indicator)
This information is very important and should be added to the geopackage log.
![Screenshot 2024-06-28 083641](https://github.com/qgis/QGIS/assets/26622829/3fec1410-fc5c-45de-b1b6-d8b7767dce76)

### Additional context

https://receiverhelp.trimble.com/alloy-gnss/en-us/NMEA-0183messages_GGA.html

**Comment by agiudiceandrea** [comment]
@geopackix, aren't also the "Mode" and "Dimensions" fields missing?

**Comment by geopackix** [comment]
@agiudiceandrea Yes. this data is also mission in the geopackage log, but not that critical.
